src/pages/write/tags/[tag].astro

```
@@ -1,96 +1,110 @@
+        return import.meta.env.PROD ? !data.draft : true;
+    });
 
-  // Get all unique tags
-  const tags = new Set<string>();
-  posts.forEach((post) => {
-    post.data.tags.forEach((tag) => tags.add(tag));
-  });
+    // Get all unique tags
+    const tags = new Set<string>();
+    posts.forEach((post) => {
+        post.data.tags.forEach((tag) => tags.add(tag));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)